### PR TITLE
fix: update pass computation epoch to improve accuracy

### PR DESCRIPTION
**Pull request by vishwa2710**

### CODE REVIEW

File: include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp [hunk 1]
```diff
@@ -203,7 +203,10 @@ class Orbit : public Trajectory
     String generateFrameName(const Orbit::FrameType& aFrameType) const;
 
     static Instant GetCrossingInstant(
```
Comment (kyle-cochran) [inline include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp:205]: nit: should the epoch be optional here?

File: include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp [hunk 2]
```diff
@@ -203,7 +203,10 @@ class Orbit : public Trajectory
     String generateFrameName(const Orbit::FrameType& aFrameType) const;
```
Comment (kyle-cochran) [inline include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp:204]:
```suggestion

    /// @brief Find the Instant at which the return value of `getValue` crosses zero.
    ///
    ///              Use a bisection search to find the Instant between `previousInstant` and `nextInstant` at which the return value of `getValue` crosses zero. 
    ///
    /// @param anEpoch An orbit epoch from which to measure
    /// @param previousInstant lower bound of search
    /// @param getValue function that receives a duration [s] from `anEpoch` and returns a real value
    /// @return Instant
```

^nit: it's not super obvious what the extra Instant is for

**Review by kyle-cochran — APPROVED** [review]

File: include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp (hunk 1 above)
Comment (vishwa2710) [reply include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp:205]: It's required to offset the instants to convert to double. So it's required, I don't want to have a fallback to J2000 or something due to the precision issues.

Comment (kyle-cochran) [reply include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp:213]: Okay. I missed that it was a private function. Disregard

**Pull request merged**